openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -132,26 +132,19 @@ def create_account_with_params(request, params):
             ]}
         )
 
-    do_external_auth, eamap = pre_account_creation_external_auth(request, params)
-
     extended_profile_fields = configuration_helpers.get_value('extended_profile_fields', [])
     # Can't have terms of service for certain SHIB users, like at Stanford
     registration_fields = getattr(settings, 'REGISTRATION_EXTRA_FIELDS', {})
     tos_required = (
         registration_fields.get('terms_of_service') != 'hidden' or
         registration_fields.get('honor_code') != 'hidden'
-    ) and (
-        not settings.FEATURES.get("AUTH_USE_SHIB") or
-        not settings.FEATURES.get("SHIB_DISABLE_TOS") or
-        not do_external_auth or
-        not eamap.external_domain.startswith(settings.SHIBBOLETH_DOMAIN_PREFIX)
     )
 
     form = AccountCreationForm(
         data=params,
         extra_fields=extra_fields,
         extended_profile_fields=extended_profile_fields,
-        do_third_party_auth=do_external_auth,
+        do_third_party_auth=False,
```

Comment:
Nit: Is this always `False`?  If so, we can add a comment that it could possibly be removed (for further refactoring later).

Reply:
Wasn't sure, but suspected it might not be